package.json

```diff
@@ -27,7 +27,7 @@
       "npm run lint-fix",
       "git add"
     ],
-    "*.{css,scss,json,md}": [travis
+    "*.{css,scss,json,md}": [
```

Comment:
Don't you think we should set up Travis CI?

Reply:
This was a typo on my part - I was trying to search "travis" and I accidentally typed it into package.json